src/filemenu/filemenu_msg.c

```diff
@@ -309,12 +306,24 @@ void filemenu_draw_message(u8* message, s32 x, s32 y, s32 alpha, s32 color, u32
         filemenu_draw_char(0xF3, x, y, flag1, color, flag2);
         tmp = message;
         while (*tmp != 0xFD) {
```

Comment:
Should use MSG_CHAR_READ_END here, likewise for other message character IDs in this function.

Reply:
This is already in the codebase, not part of the iQue changes